app/src/verifier.h

```diff
@@ -326,9 +342,7 @@ namespace scitt::verifier
           // Verify signature.
           try
           {
-            // TODO: replace with cose::verify() once the t_cose workarounds are
-            // removed.
-            cose::notary_verify(data, phdr, key);
+            cose::verify(data, key, true);
```

Comment:
```suggestion
            cose::verify(data, key, /* allow_unknown_crit */ true);
```

Also, add a short comment above saying where critical parameters are checked.